tartiflette/sdl/transformers/helpers/reduce_type.py

```diff
@@ -0,0 +1,14 @@
+from tartiflette.sdl.transformers.schema import GraphQLType, GraphQLListType, \
+    GraphQLNonNullType, GraphQLNamedType
+
+
+def reduce_gql_type(gql_type: GraphQLType) -> GraphQLNamedType:
+    if isinstance(gql_type, (GraphQLListType, GraphQLNonNullType)):
```

Comment:
What is the purpose of that class? Could you a small description with a sample?

Reply:
I'm not sure we'll keep it as is... I'll add a description if we keep it :)